lib/univalue_get.cpp

```diff
@@ -59,6 +59,22 @@ bool ParseInt64(const std::string& str, int64_t *out)
         n <= std::numeric_limits<int64_t>::max();
 }
 
+
+bool ParseUint64(const std::string& str, uint64_t *out)
+{
+    if (!ParsePrechecks(str))
+        return false;
+    char *endp = NULL;
+    errno = 0; // strtoll will not set errno if valid
```

Comment:
Copy paste typos here and below? `strtoull ... `.

Reply:
thanks, fixed